tests/unit/core/sagas/testUtils.js

```diff
@@ -6,13 +6,23 @@ import SagaTester from 'redux-saga-tester';
 import { put, select } from 'redux-saga/effects';
 /* eslint-enable import/order */
 
-import createStore from 'amo/store';
-import { setClientApp, setLang } from 'core/actions';
 import apiReducer from 'core/reducers/api';
-import { getApi } from 'core/sagas/utils';
+import authReducer from 'core/reducers/authentication';
+import { createErrorHandler, getApi, getAuth } from 'core/sagas/utils';
+import { dispatchSignInActions } from 'tests/unit/amo/helpers';
 
 
 describe('Saga utils', () => {
+  it('does not allow usage of dispatch from a saga', () => {
+    const fakeLog = { error: sinon.stub() };
+    const errorHandler = createErrorHandler('some-error-handler', {
+      log: fakeLog });
+    errorHandler.dispatch('ANYTHING');
+
+    sinon.assert.calledWith(fakeLog.error,
```

Comment:
Thanks for adding test coverage! We really need to get back up to 100% coverage so we don't miss stuff like this.